docs/src/Atmos/Model/risingbubble.md

```diff
@@ -0,0 +1,447 @@
+    At the very least you must specify the initial conditions. Detailed information
+    on subcomponent argument options can be found in their respective docs.
+
+Finally,  we pass a `Problem Name` string, the mesh information, and the model type to  the `AtmosLESConfiguration` object.
+
+```@example risingbubble
+    config = CLIMA.AtmosLESConfiguration(
+        "DryRisingBubble", # Problem title [String]
+        N,                 # Polynomial order [Int]
+        resolution,        # (Δx, Δy, Δz) effective resolution [m]
+        xmax,              # Domain maximum size [m]
+        ymax,              # Domain maximum size [m]
+        zmax,              # Domain maximum size [m]
+        param_set,         # Parameter set. Model optional but parameter set not optional.
+        init_risingbubble!,# Function specifying initial condition
+        solver_type = ode_solver, # Time-integrator type
+        model = model, # Model type
+    )
+    return config
+end
+```
+
+!!! note
+    `Keywords` are used to specify some arguments (see appropriate source files).
+
+## Diagnostics
+
+```@example risingbubble
+function config_diagnostics(driver_config)
+    interval = "10000steps"
+    dgngrp = setup_atmos_default_diagnostics(interval, driver_config.name)
+    return CLIMA.DiagnosticsConfiguration([dgngrp])
+end
+
+function main()
+    CLIMA.init()
+```
+
+Working precision
+
+```@example risingbubble
+    FT = Float64
+```
+
+DG polynomial order
+
+```@example risingbubble
+    N = 4
+```
+
+Domain resolution and size
+
+```@example risingbubble
+    Δh = FT(50)
+    Δv = FT(50)
+    resolution = (Δh, Δh, Δv)
+```
+
+Domain extents
+
+```@example risingbubble
+    xmax = FT(2500)
+    ymax = FT(2500)
+    zmax = FT(2500)
+```
+
+Simulation time
+
+```@example risingbubble
+    t0 = FT(0)
+    timeend = FT(1000)
+```
+
+Courant number
+
+```@example risingbubble
+    CFL = FT(20)
+
+    driver_config = config_risingbubble(FT, N, resolution, xmax, ymax, zmax)
+    solver_config = CLIMA.SolverConfiguration(
+        t0,
+        timeend,
+        driver_config,
+        init_on_cpu = true,
+        Courant_number = CFL,
+    )
+    dgn_config = config_diagnostics(driver_config)
+```
+
+User defined filter (TMAR positivity preserving filter)
+
+```@example risingbubble
+    cbtmarfilter = GenericCallbacks.EveryXSimulationSteps(1) do (init = false)
+        Filters.apply!(solver_config.Q, 6, solver_config.dg.grid, TMARFilter())
+        nothing
+    end
+```
+
+Invoke solver (calls solve! function for time-integrator)
+
+```@example risingbubble
+    result = CLIMA.invoke!(
+        solver_config;
+        diagnostics_config = dgn_config,
+        user_callbacks = (cbtmarfilter,),
+        check_euclidean_distance = true,
+    )
+
+    @test isapprox(result, FT(1); atol = 1.5e-3)
+end
+
+main()
+```
+
+---
+
+*This page was generated using [Literate.jl](https://github.com/fredrikekre/Literate.jl).*
```

Comment:
Is it best practice to check these into the repository or should we generate them automatically? I would tend towards the former.